ttnn/cpp/ttnn/operations/pool/maxpool/device/max_pool2d_multi_core_program_factory.cpp

```diff
@@ -168,6 +171,21 @@ MaxPoolNew::MultiCore::cached_program_t max_pool_2d_multi_core_sharded_with_halo
     auto in_tiled_cb = tt::tt_metal::CreateCircularBuffer(program, all_cores, in_tiled_cb_config);
     log_debug(tt::LogOp, "CB {} :: PS = {}, NP = {}", in_tiled_cb_id, in_tiled_cb_pagesize, in_tiled_cb_npages);
 
+    uint32_t max_pool_partials_cb_id = tt::CB::c_intermed1;  // max_pool partials
```

Comment:
can this CB creation made conditional for large window sizes only? that will keep the current mem usage same for existing models.

Reply:
Done.